openslide/deepzoom.py

```diff
@@ -213,19 +216,21 @@ def _get_tile_info(self, dz_level, t_location):
         ]
         # Round location down and size up, and add offset of active area
         l0_location = tuple(
-            int(self._l0_from_l(slide_level, l) + l0_off)
-            for l, l0_off in zip(l_location, self._l0_offset)
+            int(self._l0_from_l(slide_level, loc) + l0_off)
+            for loc, l0_off in zip(l_location, self._l0_offset)
         )
         l_size = tuple(
-            int(min(math.ceil(self._l_from_z(dz_level, dz)), l_lim - math.ceil(l)))
-            for l, dz, l_lim in zip(l_location, z_size, self._l_dimensions[slide_level])
+            int(min(math.ceil(self._l_from_z(dz_level, dz)), l_lim - math.ceil(loc)))
+            for loc, dz, l_lim in zip(
+                l_location, z_size, self._l_dimensions[slide_level]
+            )
         )
 
         # Return read_region() parameters plus tile size for final scaling
         return ((l0_location, slide_level, l_size), z_size)
 
-    def _l0_from_l(self, slide_level, l):
-        return self._l0_l_downsamples[slide_level] * l
+    def _l0_from_l(self, slide_level, location):
```

Comment:
This should probably be something like `l_coordinate`.  It's not a full location.